lib/types/cql-utils.js

```diff
@@ -137,6 +137,35 @@ function encodeTuple(value, type) {
     return [rust.ComplexType.remapTupleSupportType(newTypes), res];
 }
 
+/**
+ * Wraps UDT into format recognized by ParameterWrapper
+ * @param {Object} value
+ * @param {rust.ComplexType} type
+ * @returns {Array<rust.ComplexType|Array<Array<String, any>>>} Returns tuple: [rust.ComplexType, Array<any>]
```

Comment:
:thinking: Is the comment consistent with the code?
`Array<Array<String, any>>` seems different than `Array<any>`.